PharoTour/PharoTour.pillar

```diff
@@ -353,7 +352,7 @@ The ""System Browser"", also known as "Class Browser", is one of the key tools
 used for programming. As we shall see, there are several interesting browsers
 available for Pharo, but this is the basic one you will find in any image.
 
-@@todo TODO: factorial on number would be better. and should redo the screenshot
+@@authorToDo TODO: factorial on number would be better. and should redo the screenshot
```

Comment:
Sure. Having both sounds fine. 
(Is there instrumentation around it? A command that lists all the authorToDos, or is it just for grepping?)